examples/dato/README.md

```diff
@@ -0,0 +1,141 @@
+# Makeswift + Dato Integration
+
+This integration allows you to combine the power of Dato's content management with Makeswift's visual page building capabilities. Visually create dynamic, content-rich websites with a seamless development experience.
+
+For a detailed guide on setting up and using this integration, visit:
```

Comment:
Sounds like we aren't going to have a guide on the Makeswift site. Should we remove this?